README.md

```diff
@@ -1,48 +0,0 @@
-os-autoinst/openQA tests for openSUSE and SUSE Linux Enterprise [![Build Status](https://travis-ci.org/os-autoinst/os-autoinst-distri-opensuse.svg?branch=master)](https://travis-ci.org/os-autoinst/os-autoinst-distri-opensuse?branch=master)
```

Comment:
what the reason to delete readme ?

Reply:
Hi, something unintentional happen. I’m sorry for that, it was not intentional to delete this README file.